pkg/login/interactive_login.go

```diff
@@ -27,6 +30,13 @@ func InteractiveLogin(config *config.Config) error {
 
 	config.Profile.DeviceName = getConfigureDeviceName(os.Stdin)
 	config.Profile.TestModeAPIKey = apiKey
+	displayName, errDisplay := DisplayName(nil, stripe.DefaultAPIBaseURL, apiKey)
+
+	if errDisplay != nil {
+		return err
+	}
```

Comment:
As @tomer-stripe mentioned in a previous comment, there are some limitations to accessing one's account info depending on the api key used (some api keys are restricted and don't have all accesses, more info in the [docs](https://stripe.com/docs/keys#limit-access)), which means that sometimes, despite a successful login, retrieving the display name will error.

We are looking into resolving that on the backend, but have no timeline for now unfortunately.

Meanwhile, I'd suggest updating the logic so it handles cases when the display name can't be retrieve, instead of throwing an error.

Reply:
Sure! For this, in case there is an error in retrieving the access key, I can change it to allow the execution to flow by assigning an empty string / `could not retrieve account name` to the variable `displayName`

Reply:
I think a more conservative approach (aiming to not disrupt any existing workflow for current users) would be to save the `displayName` as `""` and have the confirmation logic handle this case.

IMO saying "could not retrieve account name" might get confusing for users, so it might just be better to have the confirmation text only mention livemode/testmode and no account name.

Reply:
Okay will proceed with setting `displayName` as "" in this case